### ISSUE TITLE
Select list with Text and Values in "New" button

### ISSUE THREAD

**Issue by GDileepG** [AllenFang/react-bootstrap-table#1301]
Hi, 
I am trying to add a select list to the table item, i find there is editable attribute which can be used, for example:
var editable = {
type: "select",
options:{
values: ["choice1", "choice2","choice3"]
}
};
The dropdown is successfully added in the form. But I cannot find an option to add the items with Text and Values for the dropdown as we do in HTML.

Is there any other key for "options" for the same? Please let me know.

**Comment by AllenFang** [comment]
Currently, we can't configure the text and value select option
```js
var editable = {
    type: "select",
    options:{
        values: ["choice1", "choice2","choice3"]
    }
};
```

I'll try to improve it, but there's a workaround now, check https://github.com/AllenFang/react-bootstrap-table/blob/master/examples/js/custom/insert-modal/custom-insert-modal-field.js#L55

it can help you to custom the field in the insert modal. let me know if you have any questions, thanks

**Comment by AllenFang** [comment]
#243

**Comment by AllenFang** [comment]
Hello @GDileepG, it fixed on `v3.3.4`, check [this](https://github.com/AllenFang/react-bootstrap-table/blob/master/examples/js/advance/edit-type-table.js#L7-L24) to learn how to configure the text and value on select option. let me know if there's any concern or issues, thanks :)
